src/components/dls/Skeleton/Skeleton.module.scss

```diff
@@ -23,18 +14,16 @@ $animationDuration: 8s;
 
 .active::before {
   border-radius: var(--border-radius-default);
-  background-image: linear-gradient(
-    270deg,
-    var(--shade-0),
-    var(--shade-1),
-    var(--shade-2),
-    var(--shade-1),
-    var(--shade-0)
-  );
-  background-size: 400% 100%; // increase background size's width to 4x to make background animate-able
-  animation: loading $animationDuration ease-in-out infinite;
-
-  content: "";
+
```

Comment:
There is a missing case that the user is choosing `System` theme and his system's theme is dark. in this case the skeletons do not show

<img width="431" alt="Screen Shot 2021-11-19 at 19 17 41" src="https://user-images.githubusercontent.com/15169499/142627074-e2e85914-2162-4b01-8da6-7b3e8fb791d6.png">

Reply:
PR https://github.com/quran/quran.com-frontend-next/pull/781 has a fix for this, we can merge it first then I will rebase this PR.